spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
```

Comment:
I would break this file up with blank lines as well. You are actually using one towards the bottom, but not anywhere else. Whatever pattern you do settle on you should be consistent about it.